### ISSUE TITLE
Grouping member plans by type

### ISSUE THREAD

**Issue by michael-scheurer** [wepublish/wepublish#486]
**Is your feature request related to a problem? Please describe.**
If a medium uses a bunch of member plans, it would be great grouping them together. This could be achieved by a simple additional field within the member plans e.g. called "type". As a developer I would love to have this member plan type also in the graphql filter of this respective endpoint.

An example of the member plan list at Bajour:

![image](https://user-images.githubusercontent.com/70939054/149603748-ab6a1b46-3c1b-4cce-bf3c-ff7e7fe6ec0e.png)

For the end user these have to be grouped by "member" and "gönner" like so. At the moment this is only possible with hacks and workarounds in the frontend:

![image](https://user-images.githubusercontent.com/70939054/149603813-72c92b2b-12b7-44c0-8487-83e736938ca4.png)

**Comment by roosnic1** [comment]
@michael-scheurer do you think this is something we could also achieve with the tag feature we already have for pages and articles?

**Comment by michael-scheurer** [comment]
Yeah. Good idea. Re-using the same logic and the same approach is always best.

**Comment by roosnic1** [comment]
@michael-scheurer i added a story in our internal backlog. You'll be notified through here once it's being implemented.
